Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReportSendEmail.cs

```diff
@@ -396,6 +395,7 @@ public override void Execute(ReportInfo RI)
             Email.SMTPUser = mVE.ValueCalculated;
             Email.Body = emailReadyHtml;
             emailReadyHtml = string.Empty;
+            Email.mHTMLReportTemplate = HTMLReportTemplate;
```

Comment:
instead of this take a field on Email with name IsBodyHTML and Set it's value based on Radio button selection.
For SMS action their is no radio button so we can set as false for SMS action